packages/components/src/index.js

```diff
@@ -80,6 +80,7 @@ export { default as Snackbar } from './snackbar';
 export { default as SnackbarList } from './snackbar/list';
 export { default as Spinner } from './spinner';
 export { default as TabPanel } from './tab-panel';
+export { default as __experimentalText } from './text';
```

Comment:
There is

`export * from './text';`

in the last line.

Reply:
Oh! I missed that. I'll update that.
I think this update feels more inline with how we've been exporting our other components 🙏

Reply:
I agree that the proposed changes make sense 👍